pages/inspect.tsx

```diff
@@ -45,6 +41,9 @@ const Home: NextPage = () => {
     class: '',
   });
 
+  let isErc725X = false;
+  let isErc725Y = false;
```

Comment:
why not use state variable anymore?

Reply:
state variable caused weird async issues. If `isErc725Y = true` then an EOA is passed, the components which do the fetching do not update fast enough so try to call functions on the EOA which causes errors